core/src/main/scala/cats/data/Validated.scala

```diff
@@ -960,6 +966,14 @@ sealed abstract private[data] class ValidatedInstances extends ValidatedInstance
         fab.leftMap(f)
     }
 
+//  implicit def catsDataApplicativeForValidated[F[_], E](implicit F: Semigroup[F[E]]): Applicative[Validated[F[E], *]] =
```

Comment:
I though that was needed, but seems to compile without it. I will uncomment it if CI fails